xrpl4j-model/src/test/java/org/xrpl/xrpl4j/model/client/accounts/AccountCurrenciesRequestParamsJsonTests.java

```diff
@@ -0,0 +1,32 @@
+package org.xrpl.xrpl4j.model.client.accounts;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.google.common.primitives.UnsignedInteger;
+import org.json.JSONException;
+import org.junit.Test;
+import org.xrpl.xrpl4j.model.AbstractJsonTest;
+import org.xrpl.xrpl4j.model.client.common.LedgerIndex;
+import org.xrpl.xrpl4j.model.transactions.Address;
+import org.xrpl.xrpl4j.model.transactions.Hash256;
+import org.xrpl.xrpl4j.model.transactions.Marker;
+
+public class AccountCurrenciesRequestParamsJsonTests extends AbstractJsonTest {
```

Comment:
I think if you create a test and don't specify `ledgerIndex` when constructing `AccountCurrenciesRequestParams`, codecov will stop yelling at you